src/components/search/locationInfo.js

```diff
@@ -0,0 +1,38 @@
+const resultToLocationInfo = (result) => {
```

Comment:
Maybe we could add some comments here so we know what these helpers do?

Reply:
Done!